package.json

```diff
@@ -226,5 +233,41 @@
       "protractor-cucumber-framework",
       "postcss-cssnext"
     ]
+  },
+  "jest": {
```

Comment:
I would rather put the config into a separate `jest.config.js` file. Then we have the flexibility of javascript over just JSON (e.g. for comments).

Reply:
couldn't make jest config to reside inside /config folder as jest coverage messes up with the rootDir, and tries to find coverage inside /config folder.